mllib/src/test/scala/org/apache/spark/ml/feature/StopWordsRemoverSuite.scala

```diff
@@ -200,4 +202,17 @@ class StopWordsRemoverSuite extends MLTest with DefaultReadWriteTest {
       s"requirement failed: Column $outputCol already exists.",
       "expected")
   }
+
+  test("SPARK-28365: Fallback to en_US if default locale isn't in available locales") {
+    val dummyLocale = Locale.forLanguageTag("test")
+    val oldDefault = Locale.getDefault()
+    Locale.setDefault(dummyLocale)
```

Comment:
not a big deal but I would do try-finally to set the local back (just in case one test failure affects others)